src/dsp_tools/excel2xml.py

```diff
@@ -1217,7 +1217,19 @@ def make_text_prop(
             **kwargs,
             nsmap=xml_namespace_map
         )
-        value_.text = val.value
+        if kwargs["encoding"] == "utf8":
+            # write the text into the tag, without validation
+            value_.text = str(val.value)
+        else:
+            # enforce that the text is well-formed XML: serialize tag, insert text, and parse it again
+            value_.text = "replace-me_8(1-)replace-me"
+            content = etree.tostring(value_, encoding="unicode")
+            content = content.replace("replace-me_8(1-)replace-me", str(val.value))
```

Comment:
Can you elaborate what's happening here? Without fully understanding it, I am inclined to say that there has to be a way to achieve this without inserting and replacing some placeholder

Reply:
Better now?